traits/trait_types.py

```diff
@@ -472,22 +447,16 @@ def create_editor(self):
 
 
 class Bool(BaseBool):
-    """ Defines a trait whose value must be a Python boolean using a C-level
-        fast validator.
+    """ Defines a trait that holds a bool and uses a C-level fast validator.
     """
 
     #: The C-level fast validator to use:
     fast_validate = bool_fast_validate
 
 
-# -------------------------------------------------------------------------------
-#  'BaseCInt' and 'CInt' traits:
-# -------------------------------------------------------------------------------
-
-
 class BaseCInt(BaseInt):
     """ Defines a trait whose value must be a Python int and which supports
-        coercions of non-int values to int.
+    coercions of non-int values to int.
```

Comment:
For these cases, we should really try to come up with a one-liner that works, followed by a fuller explanation in a separate paragraph if necessary.